index.d.ts

```diff
@@ -1,4 +1,4 @@
-declare module 'meross-cloud' {
+declare module '@spacepumpkin/meross-cloud' {
```

Comment:
whats that?

Reply:
Sorry, was deploying this from a fork and had to change namespaces. Lazily did it all from master and it confused the PR. Fixed now!